### PR TITLE
Service order basic UI

### PR DESCRIPTION
**Pull request by enkryptor**

### CODE REVIEW

File: src/app/order/order.page.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<ion-toolbar color="primary">
+    <ion-buttons slot="start">
+        <ion-button>
+            <ion-icon name="arrow-back-outline"></ion-icon>
+        </ion-button>
+    </ion-buttons>
+    <ion-title>{{ 'order.title' | translate }}</ion-title>
+</ion-toolbar>
+
```
Comment (ennaque) [inline src/app/order/order.page.html:9]: need to wrap all scrollable page content with ion-content tag

File: src/app/order/order.page.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                <ion-list-header class="fs20">
+                    {{ 'order.selected-service' | translate }}
+                </ion-list-header>
+                <app-service-details [service]="service" [master]="master"></app-service-details>
+                <app-order-details [order]="order$ | async"></app-order-details>
+            </ion-list>
+        </ion-col>
+    </ion-row>
+
+    <router-outlet></router-outlet>
```
Comment (ennaque) [inline src/app/order/order.page.html:34]: dont think its good idea to use angular router inside ionic app

File: src/app/order/order.page.html (hunk 1 above)
Comment (enkryptor) [reply src/app/order/order.page.html:9]: "There should only be one content in a single view" https://ionicframework.com/docs/api/content

File: src/app/order/order.page.html (hunk 2 above)
Comment (enkryptor) [reply src/app/order/order.page.html]: ion-router-outlet always assumes full screen, we should either get rid of it, or move header to page components

**Pull request merged**